### ISSUE TITLE
Delivery Box issue

### ISSUE THREAD

**Issue by m241dan** [DarkstarProject/darkstar#2323]
You can delbox augmented items that are "exclusive"

**Comment by Hozu** [comment]
Even to other "POL" account players? Since you are able to send certain EX items to chars on the same "POL" account, I do it all the time. But if you can send it to others, then that's a problem.

**Comment by m241dan** [comment]
Yes, it was tested between two different accounts. I would be willing to bet money this only happens on items that are made r/x by augmenting. I didn't have time to test more thoroughly but two of my players sent augmented items to each other and the r/x flag was removed when the recipient received the jtem but not the augment.

**Comment by TeoTwawki** [comment]
Shouldn't be able to mail augment items at all, last I knew on retail. Which admittedly was a long time ago. But even after the mail within account thing was added they were excluded.
